pod/perldelta.pod

```diff
@@ -383,6 +383,145 @@ with C<..._flags>-suffixed variants.  These expose a simple and consistent API
 to perform numerical or string comparison which is aware of operator
 overloading.
 
+=item *
+
+Reading the string form of an integer value no longer sets the flag C<SVf_POK>.
+The string form is still cached internally, and still re-read directly by the
+macros C<SvPV(sv)> I<etc> (inline, without calling a C function). XS code that
+already the APIs to get values will not be affected by this change. XS code
+that accesses flags directly instead of using API calls to express its intent
+I<might> break, but such code likely is already buggy if passed some other
+values, such as floating point values or objects with string overloading.
+
+This small change permits code (such as JSON serealisers) to reliably determine
```

Comment:
ITYM "serialisers" - you might be thinking of the cutely-misspelled Sereal perl module.

Reply:
Actually, since [American English is the standard for Perl’s documentation](https://perldoc.perl.org/perldocstyle#Choice-of-language:-American-English), would “serializer” not be apropos?